packages/nodes-base/credentials/SalesforceOAuth2Api.credentials.ts

```diff
@@ -10,6 +10,12 @@ export class SalesforceOAuth2Api implements ICredentialType {
 	documentationUrl = 'salesforce';
 
 	properties: INodeProperties[] = [
+		{
+			displayName: 'Grant Type',
+			name: 'grantType',
+			type: 'hidden',
+			default: 'pkce',
```

Comment:
@Joffcom wouldn't changing default break existing credentials? they had to be reconnected, right?

Reply:
@michael-radency I didn't have to reconnect my existing credential but you are right, it is possible that on refresh it errors out though. Let me see if I can think of a way to test that. Originally I was thinking about using a toggle option for PKCE but for some reason the grant type didn't like working with an expression.

Reply:
@michael-radency I connected my local env using non PKCE then used this branch and it has been running all weekend with no issues so I don't think this will break anything.